### PR TITLE
Add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by TimmyChan99**
In this PR I:

  - Create class `Person` with  the following:
    - Instance vars: `@id`, `@name`, and `@age`.
    - Constructor with `name`,  `age`, and `parent_permission` as parameter.
     - Getters for `@id`, `@name`, and `@age`.
    - Setters for `@name` and `@age`.
    - Private method `is_of_age?` that returns `true` if `@age` is greater or equal to 18 and `false` otherwise.
    - Public method `can_use_services?` that returns `true` if person is of age or if they have permission from parents.
    
 - Create class `Student` with the following:
    - Inherits from `Person`.
    - Constructor extends parent's constructor by adding `@classroom` and a parameter for it.
    - Method `play_hooky` that returns `"¯\(ツ)/¯"`.

- Create class `Teacher` with the following:
    - Inherits from `Person`.
    - Constructor extends parent's constructor by adding `@specialization` and a parameter for it.
    - Override `can_use_services?` so it always returns `true`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,73 @@
...
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
```
Comment (TedLivist) [inline README.md]: - [ ] Good job using the Microverse template. However, for proper documentation, kindly change it to be more descriptive of the project.

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, name: 'Unknown')
```
Comment (TedLivist) [inline teacher.rb]: - [ ] You are missing the `parent_permission` parameter. Kindly add it.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @TimmyChan99,

Good job so far!
✔️ Classes are in different files
✔️ Use of getter and setters
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @TimmyChan99, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**